extras/ovms-operator/config/samples/intel_v1alpha1_ovms.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: intel.com/v1alpha1
+kind: Ovms
+metadata:
+  name: ovms-sample
+spec:
+  # Default values copied from <project_dir>/helm-charts/ovms/values.yaml
+  aws_access_key_id: ""
+  aws_region: ""
+  aws_secret_access_key: ""
+  config_configmap_name: ""
+  gcp_creds_secret_name: ""
+  grpc_port: 8080
+  image_name: openvino/model_server:latest
+  log_level: INFO
+  model_name: "resnet"
+  model_path: "gs://ovms-public-eu/resnet50-binary"
+  models_host_path: ""
+  models_volume_claim: ""
+  nireq: ""
+  plugin_config: '{\"CPU_THROUGHPUT_STREAMS\":\"1"}'
```

Didn't you miss one escape?
```suggestion
  plugin_config: '{\"CPU_THROUGHPUT_STREAMS\":\"1\"}'
```